models/library.go

```diff
@@ -53,19 +63,56 @@ type Library struct {
 	// The Path to the library in the container image. Empty string when Lockfile scan.
 	// This field is used to convert the result JSON of a `trivy image` using trivy-to-vuls.
 	FilePath string
+	Digest   string
+}
+
+func trivyInit() error {
+	if err := tlog.InitLogger(config.Conf.Debug, config.Conf.Quiet); err != nil {
+		return xerrors.Errorf("Failed to init trivy logger. err: %w", err)
+	}
+
+	javadb.Init(config.Conf.TrivyCacheDBDir, config.Conf.TrivyJavaDBRepository, config.Conf.TrivySkipJavaDBUpdate, config.Conf.Quiet, ftypes.RegistryOptions{})
```

Comment:
Does `javadb.Init` have to run on both trivyInit and trivyInitJavaDB?

Reply:
Oooops! Awful bug!!